modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/impl/MetaStorageManagerImpl.java

```diff
@@ -290,17 +295,20 @@ private CompletableFuture<MetaStorageServiceImpl> initializeMetaStorage(Set<Stri
 
             var disruptorConfig = new RaftNodeDisruptorConfiguration("metastorage", 1);
 
-            CompletableFuture<? extends RaftGroupService> raftServiceFuture = metaStorageNodes.contains(thisNodeName)
+            CompletableFuture<? extends RaftGroupService> raftNodeStartedFuture = metaStorageNodes.contains(thisNodeName)
```

Comment:
Can we rename this variable? It's very similar to `raftNodeStarted`, which confuses me